tests/integ/modin/resample/test_resample.py

```diff
@@ -65,6 +65,26 @@ def test_resample_date_before_snowflake_alignment_date():
     )
 
 
+@sql_count_checker(query_count=2, join_count=1)
+def test_resample_date_general_ntz():
+    date_data = native_pd.to_datetime(
+        [
+            "2018-12-01",
```

Comment:
What happened if the index to be resampled contains tz info?
cc @sfc-gh-nkrishna

Reply:
Testcase 
```date_data = native_pd.to_datetime(
        [
            '2014-08-01 09:00:00+02:00',
            '2014-08-01 10:00:00+02:00',
            '2014-08-01 11:00:00+02:00'
        ]
    )
    eval_snowpark_pandas_result(
        *create_test_dfs({"A": np.random.randn(3)}, index=date_data),
        lambda df: df.resample("2D").min(),
        check_freq=False,
    )
```
is returning  `TypeError: Cannot subtract tz-naive and tz-aware datetime-like objects.` We may need better error messaging but we do not support resample with timezone right?

Reply:
Resample currently does not work with tz info, we get `TypeError: Cannot subtract tz-naive and tz-aware datetime-like objects.`

Reply:
@sfc-gh-lmukhopadhyay Can we add the test case you described above to `test_resample_negative` as part of this PR?